### DISCUSSION TITLE
How long should vault indexing take?

### DISCUSSION THREAD

**Discussion by aagha** [logancyang/obsidian-copilot#338] [Q&A]
My Vault has 1113 files in it and I've been sitting here for about 20 minutes:

![image](https://github.com/logancyang/obsidian-copilot/assets/5522103/17ffac97-70d3-4c84-b477-a89e445122f3)

**Comment by logancyang** [comment]
Looks like it's stuck. Sometimes OpenAI kills the call, just refresh and start over.

My vault has ~5000 notes and ~1M tokens, took me 2min.

**Reply by aagha** [reply]
Now it's saying I just have 78 notes (not 1000+) and stops at 0.

I've tried to stop/start the server and restart Obsidian multiple times:

![image](https://github.com/logancyang/obsidian-copilot/assets/5522103/2436e6c8-9dd8-4725-9db4-43d76eab94c4)

How do I see the Debug log?

**Reply by logancyang** [reply]
78 because it is running `Index (refresh) vault for QA` since it detects existing indices. You can run `Force re-index` to start from scratch.

Are you on windows? Not sure what the shortcut to open console is, on mac it's cmd + opt + I. Just ask GPT4.
